cloud-controller-manager/do/resources.go

```diff
@@ -34,62 +35,211 @@ import (
 )
 
 const (
-	controllerSyncPeriod = 1 * time.Minute
-	requestTimeout       = 10 * time.Second
+	controllerSyncTagsPeriod      = 1 * time.Minute
+	controllerSyncResourcesPeriod = 1 * time.Minute
+	requestTimeout                = 1 * time.Minute
```

Comment:
Just to confirm that you increased the request timeout to account for customers with many objects?

Reply:
Also, can we rename this one to `syncTagsTimeout` to be consistent with the naming for the periods, and to tell more easily what this timeout is for compared to `syncResourcesTimeout` below?

Reply:
Yup, exactly.